src/etc/test-float-parse/Cargo.toml

```diff
@@ -8,7 +8,24 @@ publish = false
 resolver = "1"
```

Comment:
Why is this using an old resolver?

Reply:
I just didn't update what was already there. This doesn't even need to be a workspace anymore, I just removed the block.